mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -1,13 +1,26 @@
+
+import {useParams} from "react-router-dom";
 import {HeadingLevel, Title} from "src/component/title/Title";
+import {OwnWaysTable} from "src/logic/waysTable/OwnWaysTable";
+import styles from "src/logic/userPage/UserPage.module.scss";
 
 /**
  * User page
  */
 export const UserPage = () => {
+  const {uuid} = useParams();
+
   return (
-    <Title
-      level={HeadingLevel.h2}
-      text="User page"
-    />
+    <div className={styles.container}>
+      <Title
+        level={HeadingLevel.h2}
+        text="User page"
+      />
+      <Title
+        text="Own ways"
+        level={HeadingLevel.h3}
+      />
+      {uuid ? <OwnWaysTable uuid={uuid} /> : null}
```

Comment:
uuid should not be optional, redirect to not found page if it is not exist

Reply:
Done